core/domain/wipeout_service.py

```diff
@@ -466,6 +489,29 @@ def delete_user(
     _delete_models(user_id, models.Names.LEARNER_GROUP)
 
 
+def _verify_profile_picture_is_deleted(username: str) -> bool:
+    """Verify that the profile picture is deleted.
+
+    Args:
+        username: str. The username of the user.
+
+    Returns:
+        bool. True when the profile picture is deleted else False.
+    """
+    if username is None:
+        return True
```

Comment:
Why is this needed?

Reply:
The username can be None and if that is the case then the profile picture will not be present on GCS.

Reply:
But by the mypy type definition of the argument, the argument is a str right, so None can't be passed in anyway?

Reply:
Updated the mypy type.

Reply:
After some discussions I have removed this particular check and we will only be verifying the deletion of profile picture when the user role is not mobile-learner as they can have username as None.
Thanks